resources/lang/en/events/misc/desert.json

```
@@ -0,0 +1,46 @@
+[
+  {
+    "event_id": "dst_misc_any_angrylizard",
+    "location": ["desert"],
+    "season": ["newleaf", "greenleaf", "leaf-fall"],
+    "weight": 20,
+    "event_text": "m_c was sniffing at an odd lump in the sand when suddenly an upset lizard gave {PRONOUN/m_c/object} a nip on the nose and dashed away!",
+    "m_c": {
+      "status": ["warrior", "apprentice", "kitten", "elder"]
+    }
+  },
+  {
+    "event_id": "dst_misc_war_anyapps1",
+    "location": ["desert"],
+    "season": ["any"],
+    "sub_type": ["war"],
+    "tags": ["clan:apprentice"],
+    "weight": 15,
+    "event_text": "The apprentices of the Clan are taking special training to learn how to blind their enemies with dust from the ground.",
```

```suggestion
    "event_text": "The apprentices of the Clan are taking special training to learn how to blind their enemies by kicking up dust.",
```
I think the phrasing here could consider further revision, but it's a nice premise!